### ISSUE TITLE
feat: more consistent spelling of "color"/"colour" in help output

### ISSUE THREAD

**Issue by sodiboo** [eza-community/eza#1243]
https://github.com/eza-community/eza/blob/83ca7b3bde7e7d0000953c6e342325de9558ead4/src/options/help.rs#L29-L31

The help text for `--color` and `--color-scale-mode` use different variations in spelling for their internal help text. It's not a problem that they aren't agnostic (i.e. i wouldn't expect them to say `when to use termianl colo[u]rs`), but it seems odd that they are using *different* variations. This should probably be fixed, so they use the same spelling.

Moreover, it is annoying that i can't grep for either spelling reliably. Consider replacing `--colo[u]r=WHEN` with `--color=WHEN --colour=WHEN`, or `--{color,colour}=WHEN`, or something similar that fully spells out both variants (and vice versa for the other two options of course). This ensures that grepping for either term will output all the relevant options, which may be useful to someone else in the future, saving them the few seconds of confusion i experienced prior to creating this issue

![grepping the eza help output for color and colour with american and british spellings returns two different lines. there is a third color option that isn't returned by either. above is shown all three relevant lines, which ought to have been printed by the following two invocations](https://github.com/user-attachments/assets/738035ef-db7d-4fd3-8324-53b5afecd798)
(all three invocations here should output the same three lines)

**Comment by PThorpe92** [comment]
Yeah this was actually a big thing when we moved dependencies away from `ansi-term`. The original author of `exa` had everything spelled `colour`, and changing dependencies required changing not only all the docs, but all the functions and comments for consistency.

To keep backwards compatibility, we continued to support all the old flags. But yeah at some point, I think we continue supporting them, but update all the docs and references to just use `color`.
